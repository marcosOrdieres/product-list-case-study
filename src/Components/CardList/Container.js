import styled from 'styled-components';

const Container = styled.div`
  background-color: white;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  box-shadow: rgba(12, 15, 20, 0.02) 0px 0px 0px 1px,
    rgba(12, 15, 20, 0.06) 0px 0px 1px 0px,
    rgba(12, 15, 20, 0.06) 0px 2px 2px 0px;
  border-radius: 10px;
  padding: 0px;
  margin-bottom: 10px;
`;

Container.displayName = 'CardList.Container';

export default Container;