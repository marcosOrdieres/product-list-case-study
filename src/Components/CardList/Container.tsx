import styled from 'styled-components';

const Container = styled.div`
  background-color: white;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  box-shadow: rgba(12, 15, 20, 0.2) 0px 0px 0px 1px,
    rgba(12, 15, 20, 0.6) 0px 0px 1px 0px,
    rgba(12, 15, 20, 0.6) 0px 2px 2px 0px;
  border-radius: 10px;
  padding: 0px;
  margin-bottom: 30px;
`;

Container.displayName = 'CardList.Container';

export default Container;