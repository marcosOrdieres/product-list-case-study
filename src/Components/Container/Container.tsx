import styled from 'styled-components';

export interface ContainerProps {
  children?: any;
}

const ContainerStyled = styled.div`
  display: flex;
  flex-direction:column;
  width: 100vw;
  height: 100vh;
  align-items: center;
`;

export const Conatainer: React.FC<ContainerProps> = (props) => <ContainerStyled {...props}>{props.children}</ContainerStyled>;

export default ContainerStyled;