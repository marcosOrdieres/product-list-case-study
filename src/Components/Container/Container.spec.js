import React from 'react';

import Container from './Container';

describe('Container', () => {
  it('should render and match styles', () => {
    const { asFragment } = render(<Container>Container</Container>);
    expect(asFragment()).toMatchSnapshot();
  });
});
