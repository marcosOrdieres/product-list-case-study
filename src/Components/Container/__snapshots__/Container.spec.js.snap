// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render and match styles 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ bTJjre"
  >
    Container
  </div>
</DocumentFragment>
`;
